test/execute/main.js

```diff
@@ -1626,6 +1627,95 @@ describe('execute', () => {
         })
       })
 
+      it('should handle duplicate parameter inheritance from normalized swagger specifications', function () {
+        const spec = {
+          spec: {
+            host: 'swagger.io',
+            basePath: '/v1',
+            paths: {
+              '/pet/{id}': {
+                parameters: [
+                  {
+                    name: 'id',
+                    in: 'path',
+                    type: 'number',
+                    required: true
+                  }
+                ],
+                get: {
+                  operationId: 'getPetsById',
+                  parameters: [
+                    {
+                      name: 'test',
+                      in: 'query',
+                      type: 'number'
+                    }
+                  ],
+                }
+              }
+            }
+          }
+        }
+
+        const resultSpec = normalizeSwagger(spec)
+        const warnSpy = expect.spyOn(console, 'warn')
+        const req = buildRequest({spec: resultSpec.spec, operationId: 'getPetsById', parameters: {id: 123, test: 567}})
+        expect(req).toEqual({
+          url: 'http://swagger.io/v1/pet/123?test=567',
+          headers: {},
+          credentials: 'same-origin',
+          method: 'GET'
+        })
+        expect(warnSpy.calls.length).toEqual(0)
+      })
+
+      it('should warn for ambiguous parameters in normalized swagger specifications', function () {
```

Comment:
this test verifies that when there are in fact ambiguous parameters (that is two parameters with the same name that have different 'in's) the warning is thrown.